src/bot/integrations/donationalerts.js

```diff
@@ -117,6 +119,10 @@ class Donationalerts {
     }
     return enabled
   }
+
+  async reconnect() {
+    this.socket.disconnect()
```

Comment:
are you sure this will reconnect, when you disconnect socket?

Reply:
Yes, i checked this. We have 
```
this.socket.off('disconnect').on('disconnect', () => {
      this.socket.open()
    })
```
what will do auto-reconnect